### ISSUE TITLE
[BUG] Scroll wheel input ignored

### ISSUE THREAD

**Issue by Altair-Bueno** [sbarex/QLMarkdown#69]
I cannot scroll using my mouse on the preview generated from QLMarkdown using my external mouse. Other previews work fine (PDFs, TXT, RTF,...)

### Useful information

- OS: BigSur 11.3.1
- Mouse: Razer, no additional software
- QLMarkdown 1.0
- Trackpad seems fine, but mouse input gets ignored and freezes app

### Crash report form console.app

[Markdown QL Extension_2021-05-23-210845_MacBook-Pro.txt](https://github.com/sbarex/QLMarkdown/files/6528783/Markdown.QL.Extension_2021-05-23-210845_MacBook-Pro.txt)

### qlmanage -m report

[QLmanageDump.txt](https://github.com/sbarex/QLMarkdown/files/6528784/QLmanageDump.txt)

PS: I think SourceCodeSyntaxHighlight has this same bug, but console.app doesn't report anything

**Comment by sbarex** [comment]
My application does not handle the mouse in any way. 

Scrolling is delegated to the webkit view. I'm afraid this is one of the numerous webkit view problems within the Quick Look preview window. With Apple magic mouse I have not found any problems.

**Comment by Altair-Bueno** [comment]
There is no point of keeping this issue as open, already solved on Source Code Syntax Highlight

# Solution

If anyone has this same issue, check the full discussion [here](https://github.com/sbarex/SourceCodeSyntaxHighlight/issues/84).

**TLDR**: Update to macOS 12 Monterey
